package com.example.cursojetpackcomposearistidevs.components.layout

import androidx.compose.foundation.background
import androidx.compose.foundation.layout.Arrangement
import androidx.compose.foundation.layout.Row
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.graphics.Color

@Composable
fun MyRow(modifier: Modifier = Modifier) {
    Row(modifier = modifier.fillMaxSize(), horizontalArrangement = Arrangement.SpaceAround) {
        Text("Hola 1", modifier = Modifier.background(Color.Red).weight(1f))
        Text("Hola 1", modifier = Modifier.background(Color.Blue).weight(1f))
        Text("Hola 1", modifier = Modifier.background(Color.Cyan).weight(1f))
    }
}